obp-api/src/main/scala/code/model/dataAccess/AuthUser.scala

```diff
@@ -732,13 +732,13 @@ import net.liftweb.util.Helpers._
             // assuming that user's email is always validated
             logger.debug("external user "+ sub + " does not exist locally, creating one")
             AuthUser.create
-              .firstName(sub)
+              .firstName(name.getOrElse(sub))
               .email(email.getOrElse(""))
               .username(sub)
               // No need to store password, so store dummy string instead
               .password(generateUUID())
               .provider(iss)
-              .validated(true)
+              .validated(emailVerified.getOrElse("false").toLowerCase() == "true")
```

Comment:
Not need get default value and toLowerCase, like follow:
`.validated(emailVerified.exists(_.equalsIgnoreCase("true")))`

Reply:
variable `emailVerified` is a `string and it can be `True` as well

Reply:
Method String#equalsIgnoreCase already deals with the case.
This modification is no problem, no big deal. This is just another thinking that when we want to check the value of Option, exists method is also Ok, instead of getOrElse.

Reply:
@hong-wei I pushed the change to my repo.